packages/addresses/src/addresses.helpers.ts

```diff
@@ -134,38 +132,35 @@ export const isNetworkName = (str: string): str is NetworkName =>
 export const isZetaNetworkName = (str: string): str is ZetaNetworkName =>
   isZetaLocalnet(str) || isZetaTestnet(str) || isZetaMainnet(str);
 
-const MissingZetaNetworkError = new Error(
-  "ZETA_NETWORK is not defined, please set the environment variable (e.g.: ZETA_NETWORK=athens <command>)"
-);
-
-export const getAddress = (
-  address: ZetaAddress,
-  {
-    customNetworkName,
-    customZetaNetwork
-  }: { customNetworkName?: NetworkName; customZetaNetwork?: ZetaNetworkName } = {}
-): string => {
-  const { name: _networkName } = network;
-  const networkName = customNetworkName || _networkName;
+const InvalidNetworkError = new Error("NETWORK is invalid, please provide a valid value");
```

Comment:
I think it would be cool to do:
```typescript
const getInvalidNetworkError = (network: string) => new Error(`Network: ${network}. (...)`);
// ...
throw getInvalidNetworkError("an invalid network");
```

Reply:
done